opentasks/src/main/java/org/dmfs/tasks/QuickAddDialogFragment.java

```diff
@@ -77,9 +79,10 @@
     /**
      * The maximum time to add for the first time the "Task completed" info is shown.
      */
-    private final static int COMPLETION_DELAY_MAX = 1500; // ms
+    private final static int COMPLETION_DELAY_MAX = 1000; // ms
```

Comment:
I'd like to keep this delay a bit longer so new users will notice the confirmation. Please note that the actual delay gets shorter with every confirmation shown.

Reply:
Oh, I did not notice that, that's fine